src/Formatter/LegacyFormatter.php

```diff
@@ -79,22 +45,18 @@ public function __construct($options = [])
      * Sets the data (done by the parser)
      *
      * @param array $settings
-     *
-     * @return \BrowscapPHP\Formatter\PhpGetBrowser
      */
-    public function setData(array $settings)
+    public function setData(array $settings) : void
     {
         $this->settings = $settings;
-
-        return $this;
     }
 
     /**
      * Gets the data (in the preferred format)
      *
      * @return \stdClass
      */
-    public function getData()
+    public function getData() : ?\stdClass
```

Comment:
Should this one be nullable? Seems that it always returns the `$output` var (which is a stdClass) no?